frontend/src/components/MainNavigation.vue

```diff
@@ -429,6 +496,40 @@ export default {
         }
       }
     }
+
+    .project-menu {
+      border-radius: 0;
+
+      >>>.v-card {
+        border-radius: 0;
+
+        .project-filter {
+          align-items: center
+        }
+
+        .project-add > div {
+          justify-content: left;
+        }
+
+        .project-list {
+          height: auto;
+          max-height: (4 * 54px) + (2 * 8px);
+          overflow-y: auto;
+          max-width: 300px;
+
+          div > a {
+            height: 54px;
+          }
+          .project-owner {
+            font-size: 11px;
+          }
+
+          .v-list__tile--highlighted {
```

Comment:
fragile

Reply:
Yes, this is why I added a test that checks for this class.